src/hf_hydrodata/model/grid.csv

```diff
@@ -5,4 +5,5 @@ conus2_wtd,WTD 30m grid 1k,"[10, 3484, 5940]","[21.8170599154073, -126.887556928
 conus2_wtd.100,WTD 30m grid 100m,"[10, 34832, 59400]","[21.8170599154073, -126.88755692881833, 53.20274381640737, -64.7677149695924]",100,,"[-2848561.29, -1724574.36]", +proj=lcc +lat_1=30 +lat_2=60 +lon_0=-97.0 +lat_0=40.0000076294444 +a=6370000.0 +b=6370000
 conus2_wtd.30,WTD 30m grid 30m,"[10, 144287, 246056]","[21.8170599154073, -126.88755692881833, 53.20274381640737, -64.7677149695924]",24.14076631,,"[-2848561.29, -1724573.11]", +proj=lcc +lat_1=30 +lat_2=60 +lon_0=-97.0 +lat_0=40.0000076294444 +a=6370000.0 +b=6370000
 conus2.250,conus 2,"[10, 3256, 4442]","[21.8170599154073, -126.88755692881833, 53.20274381640737, -64.7677149695924]",250,,"[-2208000.30881173, -1668999.65483222]",
-smapgrid,SMAP Grid,"[1, 646, 335]","[21.8170599154073, -126.88755692881833, 53.20274381640737, -64.7677149695924]",9000,,"[-2208000.30881173, -1668999.65483222]",
\ No newline at end of file
+smapgrid,SMAP Grid,"[1, 646, 335]","[21.8170599154073, -126.88755692881833, 53.20274381640737, -64.7677149695924]",9000,,"[-2208000.30881173, -1668999.65483222]",
+"",No grid,,,,,,

```

Comment:
What is this `No Grid` entry for?

Reply:
This is because there are point observation entries in the data catalog that have a null grid. 
In the SQL data catalog this causes referential integrity checks to fail if the grid table does not have a null grid.
So I added a null grid so the join to data_catalog_entry table does not fail referential integrity.